src/docs/goose-book/src/config/defaults.md

```diff
@@ -20,7 +20,7 @@ The following defaults can be configured with a `&str`:
  - html-formatted report file name: `GooseDefault::ReportFile`
  - requests log file name: `GooseDefault::RequestsFile`
  - requests log file format: `GooseDefault::RequestsFormat`
```

Comment:
This should be [`GooseDefault::RequestFormat`](https://docs.rs/goose/*/goose/config/enum.GooseDefault.html#variant.RequestFormat)